src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -70,7 +70,6 @@
       <DesignTime>True</DesignTime>
       <DependentUpon>SqmOleCommandTarget.tt</DependentUpon>
     </Compile>
-    <Compile Include="Telemetry\SonarLintIntegrationPackage.IOleCommandTarget.cs" />
```

Comment:
Partial package class that was responsible for initialization of SQM. Moved that responsibility into the MEF service since we may need to use the MEF service even before VSIX package is initialized (team explorer is MEF driven).